docs/management/numeral.asciidoc

```diff
@@ -0,0 +1,175 @@
+[[numeral]]
+== Elastic Numeral Formatting
+
+Number formatting in {kib} is done through a pattern syntax that {kib}
```

Comment:
Title uses "Numeral formatting" and first line uses "Number formatting".  Should be consistent.